### PR TITLE
fix tapicero & webapp logs: remove heartbeat log check, move to /var/log...

### PR DESCRIPTION
**Pull request by elijh**
.../tapicero, fix webapp logwatch location.

### CODE REVIEW

File: puppet/modules/tapicero/manifests/init.pp [hunk 1]
```diff
@@ -133,4 +133,17 @@
...
+
+  augeas {
+    'logrotate_tapicero':
+      context => '/files/etc/logrotate.d/tapicero/rule',
+      changes => [ 'set file /var/log/leap/tapicero*.log', 'set rotate 7',
+                   'set schedule daily', 'set compress compress',
+                   'set missingok missingok', 'set ifempty notifempty',
+                   'set copytruncate copytruncate' ]
+  }
 }
```
Comment (varac) [inline puppet/modules/tapicero/manifests/init.pp]: how does /var/log/leap/webapp.log get populated ? i only see a rsysolg config for tapicero, not for the webapp

File: puppet/modules/site_check_mk/manifests/agent/tapicero.pp [hunk 2]
```diff
@@ -17,10 +16,5 @@
         'rm /files/etc/check_mk/mrpe.cfg/Tapicero_Procs',
         'set Tapicero_Procs "/usr/lib/nagios/plugins/check_procs -w 1:1 -c 1:1 -a tapicero"' ],
       require => File['/etc/check_mk/mrpe.cfg'];
-    'Tapicero_Heartbeat':
-      incl    => '/etc/check_mk/mrpe.cfg',
-      lens    => 'Spacevars.lns',
-      changes => 'set Tapicero_Heartbeat \'/usr/local/lib/nagios/plugins/check_last_regex_in_log -f /var/log/syslog -r "tapicero" -w 300 -c 600\'',
-      require => File['/etc/check_mk/mrpe.cfg'];
   }
```
Comment (varac) [inline puppet/modules/site_check_mk/manifests/agent/tapicero.pp]: i really would like to use the tapicero heartbeat a bit longer, so we are really sure that tapicero doesn't silently stop working.

File: puppet/modules/tapicero/manifests/init.pp (hunk 1 above)
Comment (elijh) [inline puppet/modules/tapicero/manifests/init.pp]: the webapp log file was already moved, but the logwatch stuff was not properly updated to follow it. this commit fixes that.

File: puppet/modules/site_check_mk/manifests/agent/tapicero.pp (hunk 2 above)
Comment (elijh) [inline puppet/modules/site_check_mk/manifests/agent/tapicero.pp]: unless the tests are running frequently, it is not a bug for there to be no tapicero activity.

Comment (elijh) [inline puppet/modules/site_check_mk/manifests/agent/tapicero.pp]: i mean: it is normal for there to be no tapicero activity, so this check creates a lot of false positives. however, if the tests are running at a frequency that is less than the tapicero logcheck heartbeat, then in this case the logcheck does perform a benefit.

File: puppet/modules/tapicero/manifests/init.pp (hunk 1 above)
Comment (elijh) [inline puppet/modules/tapicero/manifests/init.pp]: in site_webapp/manifests/logging.pp

    rsyslog::snippet { '01-webapp':
      content => 'if $programname == "webapp" then /var/log/leap/webapp.log
    &~'
    }

**Comment by elijh** [comment]
this pr is superseded by this one: https://github.com/leapcode/leap_platform/pull/70

**Pull request closed without merging**